build/yaml/botbuilder-dotnet-sign.yml

```diff
@@ -33,7 +33,7 @@ variables:
   TestConfiguration: Release
   BuildPlatform: any cpu
   IsBuildServer: true # Consumed by projects in Microsoft.Bot.Builder.sln.
-  MSBuildArguments: -p:PublishRepositoryUrl=true -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
+  MSBuildArguments: -p:PublishRepositoryUrl=true -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg -p:RunAnalyzersDuringBuild=false
```

Comment:
I am not sure about disabling check on signed builds, normally these are executed nightly and weekly and disabling check could potentially create a loophole if an admin forces a merge.
Have quantified how much time are we cutting by disabling this check? (I am OK with disabling the check for fn test builds)